device-types/Cable Matters/180013.yaml

```diff
@@ -0,0 +1,61 @@
+---
+manufacturer: Cable Matters
+model: 24 Port Keystone Patch Panel
+slug: cable-matters-24-port-keystone-patch-panel
+# Cable Matters appears to use sequential model numbers.
+# I am using a generic but appropriate model number here
+# The part number matches their website
+# # https://www.cablematters.com/pc-386-162-rack-or-wall-mount-24-port-blank-patch-panel.aspx
+part_number: '180013'
+u_height: 1
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
If you have any questions, please let us know :)